[2025-06-14T16:12:25.566+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-14T16:12:25.733+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: globalsuperstore.extract.extract_and_load_to_staging manual__2025-06-14T16:11:57.124591+00:00 [queued]>
[2025-06-14T16:12:25.748+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: globalsuperstore.extract.extract_and_load_to_staging manual__2025-06-14T16:11:57.124591+00:00 [queued]>
[2025-06-14T16:12:25.749+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-14T16:12:25.773+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract.extract_and_load_to_staging> on 2025-06-14 16:11:57.124591+00:00
[2025-06-14T16:12:25.781+0000] {standard_task_runner.py:63} INFO - Started process 3479 to run task
[2025-06-14T16:12:25.786+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'globalsuperstore', 'extract.extract_and_load_to_staging', 'manual__2025-06-14T16:11:57.124591+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/global_superstore_dw_etl.py', '--cfg-path', '/tmp/tmptsmlty1r']
[2025-06-14T16:12:25.789+0000] {standard_task_runner.py:91} INFO - Job 65: Subtask extract.extract_and_load_to_staging
[2025-06-14T16:12:25.839+0000] {task_command.py:426} INFO - Running <TaskInstance: globalsuperstore.extract.extract_and_load_to_staging manual__2025-06-14T16:11:57.124591+00:00 [running]> on host f3cca4675eb8
[2025-06-14T16:12:25.968+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='globalsuperstore' AIRFLOW_CTX_TASK_ID='extract.extract_and_load_to_staging' AIRFLOW_CTX_EXECUTION_DATE='2025-06-14T16:11:57.124591+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-14T16:11:57.124591+00:00'
[2025-06-14T16:12:25.970+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-14T16:12:26.037+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-06-14T16:12:26.050+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T16:12:26.330+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T16:12:26.973+0000] {logging_mixin.py:188} INFO - ✔ Đã load Category → staging.stg_Category
[2025-06-14T16:12:26.980+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T16:12:26.996+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T16:12:31.185+0000] {logging_mixin.py:188} INFO - ✔ Đã load Customer → staging.stg_Customer
[2025-06-14T16:12:31.200+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T16:12:31.215+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T16:12:44.115+0000] {logging_mixin.py:188} INFO - ✔ Đã load Order_Details → staging.stg_Order_Details
[2025-06-14T16:12:44.125+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T16:12:44.142+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T16:12:48.793+0000] {logging_mixin.py:188} INFO - ✔ Đã load Orders → staging.stg_Orders
[2025-06-14T16:12:48.799+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T16:12:48.805+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T16:12:49.629+0000] {logging_mixin.py:188} INFO - ✔ Đã load Product → staging.stg_Product
[2025-06-14T16:12:49.638+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T16:12:49.646+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T16:12:49.698+0000] {logging_mixin.py:188} INFO - ✔ Đã load Sub_Category → staging.stg_Sub_Category
[2025-06-14T16:12:49.698+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-14T16:12:49.699+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-14T16:12:49.711+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=globalsuperstore, task_id=extract.extract_and_load_to_staging, run_id=manual__2025-06-14T16:11:57.124591+00:00, execution_date=20250614T161157, start_date=20250614T161225, end_date=20250614T161249
[2025-06-14T16:12:49.793+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-14T16:12:49.856+0000] {taskinstance.py:3503} INFO - 6 downstream tasks scheduled from follow-on schedule check
[2025-06-14T16:12:49.860+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
